pytest.ini

```diff
@@ -0,0 +1,6 @@
+# pytest.ini
```

Comment:
This is a good direction to resolve the challenge but we prefer to keep the number of files small so this should be configured in `pyproject.toml` instead. See [pytest docs on config using `pyproject.toml`](https://docs.pytest.org/en/stable/customize.html#pyproject-toml).